modules/installation-configuration-parameters.adoc

```diff
@@ -761,6 +765,14 @@ host must trust the certificate.
 |If you provide the VPC instead of allowing the installation program to create the VPC for you, specify the subnet for the cluster to use. The subnet must be part of the same `machineNetwork[].cidr` ranges that you specify. For a standard cluster, specify a public and a private subnet for each availability zone. For a private cluster, specify a private subnet for each availability zone.
 |Valid subnet IDs.
 
+|`compute.platform.aws.rootVolume.kmsKeyARN`
+|The Amazon Resource Name (key ARN)) of a KMS key. This is required to encrypt cluster worker nodes OS volumes with specific KMS key.
+|Valid link:https://docs.aws.amazon.com/kms/latest/developerguide/find-cmk-id-arn.html[key ID or the key ARN].
+
+|`controlPlane.platform.aws.rootVolume.kmsKeyARN`
```

Comment:
@xenolinux can you provide preview link here?  I suppose this section should be after `*.platform.aws.rootVolume.type`, thanks

Reply:
@yunjiang29 @Phaow I added the preview link. Please take a look.

Reply:
> @yunjiang29 @Phaow I added the preview link. Please take a look.

Hi @xenolinux , I guess @yunjiang29  suggested that the `kmsKeyARN` parameter description better after `rootVolume.type`. But seems I couldn' find the `controlPlane.platform.aws.rootVolume.type`parameter.
![image](https://user-images.githubusercontent.com/86950592/183579599-73d30935-d9d9-407a-b907-cee2008801bd.png)

Reply:
@Phaow  I added the fileds after `compute.platform.aws.rootVolume.type` if that's okay.

Reply:
Thanks! looks good to me. But seems the doc also miss the `controlPlane.platform.aws.rootVolume.type` parameter.
Let's wait @yunjiang29 's reply to avoid we missed someting else.